### DISCUSSION TITLE
Schneider Electric MEG5126 usable stand alone?

### DISCUSSION THREAD

**Discussion by d-sko** [Koenkk/zigbee2mqtt#18019] [Devices]
Hello,

I'm a bit unclear about the MEG5126 (https://www.zigbee2mqtt.io/devices/MEG5126-0300.html). Can this thing be used "stand alone" kind of as a remote just to get the button presses via zigbee2mqtt or can it only be used connected to one of the "PlusLink"-actors? and if it is usable standalone: how is it powered?

**Comment by linuxrrze** [comment]
It can only be connected to the Pluslink actors (it's just plugged in).
However I tried to get it working, but for now I can only use one of the buttons (the other one does not trigger the power relay).
Not sure wether I just got a broken one or if it's really a general problem (as Schneider Electric's support told me).

**Reply by linuxrrze** [reply]
A valid combination should be:
MEG5162-0000
MEG5126-0300 (System M) / MEG5126-6000 (System Design)
